examples/custom-types/src/lib.rs

```diff
@@ -1,98 +1,70 @@
 use url::Url;
 
 // A custom guid defined via a proc-macro (ie, not referenced in the UDL)
-// By far the easiest way to define custom types.
 pub struct ExampleCustomType(String);
+
+// custom_newtype! is the easiest way to define custom types.
```

Comment:
Also, `custom_newtype!` is not obvious what it does from the name. But I think this one is used throughout the codebase.